### DISCUSSION TITLE
"detail": "Authentication credentials were not provided."

### DISCUSSION THREAD

**Discussion by SeaForeEx** [bpepple/metron#176] [Q&A] [answered]
Trying to test API calls in Postman, I ran the call ```https://metron.cloud/api/publisher/``` and got the following message in Postman:

```
{
    "detail": "Authentication credentials were not provided."
}
```

I'm not sure how to provide these credentials, the request headers for this call have me confused.

I'm going through a Bootcamp trying to figure this thing out, any help would be appreciated.

**Comment by bpepple** [comment, marked as answer]
Yeah, you need to add your credentials to Postman for the calls to work.

I believe you can find the info to do that here:

https://www.baeldung.com/java-postman-authentication

If you still problems setting it up, I can help out tomorrow.

**Reply by SeaForeEx** [reply]
This link helped me out, just had to take a second to remember my password.

Get ready for more questions, I'm on here!
